packages/client/src/utils/gateway.ts

```diff
@@ -1008,6 +1008,9 @@ export type Identifier = {
 
 export enum IdentityIdType {
```

Comment:
@Zangetsu101 I dont think these are required anymore as any id type can be sent now?

Reply:
yeah I've made them normal strings now in the mapping functions